tests/python/relax/test_op_index.py

```diff
@@ -1010,5 +1011,81 @@ def strided_slice(
     tvm.ir.assert_structural_equal(expected, after)
 
 
+def test_legalize_dynamic_begin_inf_end():
+    """relax.op.strided_slice FLegalize must support dynamic begin/end"""
+
+    @I.ir_module
+    class before:
```

Comment:
You can use `# fmt: off` and `# fmt: on` to disable black for specific code (e.g. TVMScript)